src/pathfinding_service/database.py

```diff
@@ -53,27 +63,49 @@ def upsert_iou(self, iou: IOU) -> None:
             """
             INSERT OR REPLACE INTO iou (
                 sender, amount, expiration_block, signature, claimed
-            ) VALUES (:sender, :amount, :expiration_block, :signature, :claimed)
+            ) VALUES (
+                :sender, printf('0x%064x', :amount), printf('0x%064x', :expiration_block),
```

Comment:
Is `printf` a sqlite function?

Reply:
Yes, it behaves just like the C `printf`.